src/deployment/flavors.md

```diff
@@ -3,41 +3,158 @@ title: Creating flavors for Flutter
+
+## Using flavors in iOS
+
+Now that you’ve set up your development flavor, 
+you can add different product bundle identifiers per flavor. 
+A bundle identifier is a way to uniquely identify your application. 
+In this example, we want the Debug-development value to equal 
+com.flavor-test.dev. 
+
+1. Change the app bundle identifier to differentiate between schemes. 
+In **Product Bundle Identifier**, append `.dev` to each -development scheme value.            
+![Step 1 using flavors image.](/assets/images/docs/flavors/step1-using-flavors.png){:width="100%"}        
+2. In the Build Settings, set the Product Name value to match each flavor.        
+![Step 2 using flavors image.](/assets/images/docs/flavors/step2-using-flavors.png){:width="100%"}             
+3. Add the display name to info.plist.        
+Create a new key in info.plist called **Bundle Display Name** 
+with the value `$(PRODUCT_NAME)`. 
+![Step 2 using flavors image.](/assets/images/docs/flavors/step2-using-flavors.png){:width="100%"}    
+
+Now you have set up your development flavor by making a `development` scheme 
+in Xcode and setting the build configurations for that scheme. 
+
+For more information, skip to the [“Launching your app flavors”][] 
+section at the end of this document.
+
+## Using flavors in Android
+
+Setting up flavors in Android can be done in your projects 
+**build.gradle** file.
+
+1. In your IDE, inside your Flutter project, 
+navigate to **android** > **app** > **build.gradle**.   
+
+![Step 1 using flavors in Android image.](/assets/images/docs/flavors/android-step1.png){:width="100%"}    
+
+Add a flavors object with the specified environments along with values for 
+dimension, resValue, and applicationId. 
+The name of the application for each build is located in resValue 
+(as shown in the screenshot below from VSCode).
+## Setting up launch configurations
+
+If you are using VS Code as your IDE, the following steps 
+show you how to set up launch configurations. 
+Adding a **launch.json** file allows you to run the terminal command 
+`flutter run --flavor [environment name]`. 
+1. In the root directory of your project, add a folder called **.vscode**.    
+2. Inside the **.vscode** folder, create a file named **launch.json**.    
+3. In the **launch.json** file, add a configuration object for each flavor. 
+Each configuration has a **name**, **request**, **type**, **program**, 
+and **args** key.
+
+![Image for VSCode launch configuration step.](/assets/images/docs/flavors/vscode-config.png){:width="100%"}  
+
+## Launching your app flavors
+
+Once you’ve set up your flavors in iOS and/or Android, 
+make modifications to your Dart code in **lib** > **main.dart** 
+that allow you to consume the flavors. 
+Run `flutter run --flavor [environment name]` in your terminal. 
+Check out the [Flutter repo][] for integration test examples of 
+build flavors for [iOS(Xcode)][] and [Android][]. 
```

Comment:
This section is confusing to me. These are steps, but they aren't really called out as steps. Where do they see *how* to modify the Dart code?

I would suggest:
```suggestion
1. Once the flavors are set up, modify the Dart code in
    **lib** > **main.dart**  to consume the flavors.
    
   [insert code block example - not a screen shot] 

2. Test the setup using `flutter run --flavor [environment name]`
    at the command line, or by setting up a run configuration in your IDE.
    
For examples of build flavors for [iOS(Xcode)][] and [Android][],
check out the integration test samples in the [Flutter repo][].
```